include/kodlab_mjbots_sdk/behavior_manager.h

```diff
@@ -0,0 +1,295 @@
+/**
+ * @file behavior_manager.h
+ * @author Ethan J. Musser (emusser@seas.upenn.edu)
+ * @brief Container for managing the `Behaviors` available to a robot and
+ *        switching between them
+ * @date 7/26/22
+ * 
+ * @copyright Copyright 2022 The Trustees of the University of Pennsylvania. All
+ *            rights reserved.
+ * 
+ */
+
+#pragma once
+
+#include <iostream>
+#include <memory>
+#include "kodlab_mjbots_sdk/robot_base.h"
+#include "kodlab_mjbots_sdk/behavior.h"
+#include "kodlab_mjbots_sdk/off_behavior.h"
+#include "kodlab_mjbots_sdk/log.h"
+
+namespace kodlab
```

Comment:
lets think about making a name space and possibly a separate folder for behavior related code. It will make it easier to share the code you wrote when you apply for jobs.

Reply:
Sounds good to me!  If it's alright with you, I will wait and do this in a separate PR once #40 and #41 are merged since we have already merged #39 (`Behavior` and `IOBehavior`) in the current `kodlab` namespace.

Reply:
An issue has been opened and assigned (#57).